src/lib/utils/simulator.ts

```diff
@@ -57,6 +78,8 @@ export class TxSimulator {
       throw Error('Tenderly simulation quota reached')
     }
     const data = await res.json()
+    console.log('ten', data.simulation.id, data.simulation.shared)
+    this.share(data.simulation.id)
```

Comment:
Avoiding await to never make the result wait for this nice-to-have.

Obsolete logs will be removed before merging.